W-14221349 Pdk troubleshooting init (#391)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc

For more information about PDK's debugging environment, see xref:policies-pdk-debug-local.adoc[].

[[debug-logs]]
== Debug in a Production Environment

> It is possible that errors that are not present in PDK's debugging environment are present when the policy is deployed to a production environment.

Reviewer: ```suggestion
Errors that are not present in the PDK debugging environment might be present when the policy is deployed to a production environment.
```

Reviewer: Don't use possessives with product/feature names. 

Is PDK an approved initialism?
